<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informe o diâmetro dos condutores</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="btnDialog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnidades">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidades</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbUnidades">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>m</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblDiametro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diâmetro</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbDiametro">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000500000000000</double>
    </property>
    <property name="value">
     <double>19.050000000000001</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="editArquivo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblArquivo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivo</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnArquivo">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>31</y>
      <width>27</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
